### PR TITLE
Fix for failing CI build

### PR DESCRIPTION
**Pull request by bpathak-ons**
NOTE - Code vulnarabilities are supressed for CVSS between 7-8(Critical). This is raised as a separate bug task in the backlog (by kapow team lead as I don't have access to the board yet)

What
* Fix for failing CI build

How to review
* CI build pass

Who
* anyone not me

### CODE REVIEW

File: ci/build.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-java
+
+inputs:
+  - name: dp-logging
+
+outputs:
+  - name: build
+
```
Comment (cookel2) [inline ci/build.yml:15]: You may need to add this here:
```suggestion
caches:
  - path: go/
  
```

File: ci/build.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+run:
+  path: dp-logging/ci/scripts/build.sh
```
Comment (cookel2) [inline ci/build.yml:17]: new line needed here

File: ci/scripts/audit.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+pushd dp-logging
+    make audit
+popd
```
Comment (cookel2) [inline ci/scripts/audit.sh:5]: new line needed here

File: ci/audit.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-java
+
+inputs:
+  - name: dp-logging
+
+run:
+  path: dp-logging/ci/scripts/audit.sh
```
Comment (cookel2) [inline ci/audit.yml:13]: new line needed here

**Review by cookel2 — CHANGES_REQUESTED** [review]
There are a few places where a new line is needed (formatting) and I've added a suggestion for something to the build.yml

File: ci/scripts/audit.sh (hunk 3 above)
Comment (bpathak-ons) [reply ci/scripts/audit.sh:5]: new line won't make any difference

File: ci/build.yml (hunk 1 above)
Comment (bpathak-ons) [reply ci/build.yml:15]: this is a java service so not needed as per other exisiting java services

**Comment by daiLlew** [comment]
I think we may also need to add a `Makefile` to this repo. The CI checks are still failing because its expecting `make audit` and `make test` profiles. Take a look at the Zebedee project and copy/edit that `Makefile` and add it to this repo.

**Comment by cookel2** [comment]
I think you need to create a .nancy-ignore file (for example, like the one in dp-search-reindex-api) that lists the various CVE numbers for the vulnerabilities that are preventing the audit tests from passing. E.g. try adding these ones: CVE-2020-27216, CVE-2021-28165, CVE-2020-27223, CVE-2021-34428, CVE-2020-27218, CVE-2021-28169.

File: ci/lint.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.15.7
```
Comment (daiLlew) [inline ci/lint.yml]: I think this should be `onsdigital/dp-concourse-tools-java` not golang

**Review by cookel2 — DISMISSED** [review]
looks good

File: ci/lint.yml (hunk 5 above)
Comment (bpathak-ons) [reply ci/lint.yml]: ok, now changed to latest image for java

**Review by daiLlew — APPROVED** [review]

**Pull request merged**